index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
 
   </section>
 
+  <section class="appendix">
+    <h1>What Does a DID Identify?</h1>
+    <p>
+  The DID Core specification states that a <a>DID</a> identifies the
+  <a>DID subject</a>—and that a <a>DID subject</a> can be anything that can be
+  identified with a URI. However some applications of <a>DIDs</a>, particularly
+  those involving the Semantic Web, may need a more precise definition of how
+  DID identification works. That is the purpose of this Appendix.
```

Comment:
```suggestion
  identified with a URI. This Appendix provides additional details about how <a>DIDs</a> 
  work with respect to identification and description.
```

I don't think we need the bit about Sem Web here, I suspect it's nice for about 2% of readers, meaningless to 70% and carries unpleasant baggage for the rest.